developer_guide/installation/installation_archive.rst

```diff
@@ -18,6 +18,13 @@ If you prefer, you can also download them directly from the command line:
     $ wget http://download.akeneo.com/pim-community-standard-v1.5-latest-icecat.tar.gz #for icecat version
     $ wget http://download.akeneo.com/pim-community-standard-v1.5-latest.tar.gz #for minimal version
 
+Downloading the Enterprise Edition (EE) archive
```

Comment:
the CE and EE archives are not the same
CE archive contains vendors
not the EE one

so the installation procedure is not stricly the same

I'm wondering if we shouldn't split this installation into 2 distinct pages, one for CE and one for EE (even if we can mutualise a lot of things)